src/test/resources/harness-config.yml

```diff
@@ -2,6 +2,13 @@ inputFormat: xml
 context: testContext
 
 databasesUnderTest:
+  - name: informix
```

Comment:
duplicated entry, we have 14 and 12 at the end of the file

Reply:
Nice catch!